tests/integration/README.md

```diff
@@ -218,6 +219,13 @@ In case you want to debug the utility script itself, there is the verbose mode f
 * `result` (`string`, required): The output file (a relative path from test case directory).
 * `expected` (`string`, required): The expected output file (a relative path from test case directory).
 
+### Vallidate Schema step options Schema
+
+* `kind` (`"json_schema"`, required): The kind of schema validation to perform. For now, only json-schema is supported.
+* `result` (`string`, required): The output file (a relative path from test case directory).
+* `schema_type` (`output_json_report`, required): The type of schema for the validation. These are the default schemas available for the integration test.
+* `custom_schema_path` (`string`, optional): The path to the custom schema (a relative path from test case directory). If it is provided, the validation will use this schema and ignore the schema corresponding with `schema_type`.
+
```

Comment:
I think `schema_type` should instead be `schema_name` or maybe just `schema`, since it refers to a particular schema rather than a "type" of schema (`schema_type` sounds more like what `kind` is for).

Also, there is a typo in `Vallidate` (-> `Validate`).

Reply:
I have changed it to `schema` and fix the typo error in this commit - https://github.com/oracle/macaron/pull/824/commits/c56aabd62518f5fe2a27740f6fcb2f8f326ab7be